chart.js

```diff
@@ -1,18 +1,34 @@
 'use strict';
 
-function runChart(){
-var ctx = document.getElementById('chart').getContext('2d');
-var numberData = finalClicks;
-var labelColors = finalNames;
+var numberData = [];
+var objectNames = [];
 
+var allData = [];
+
+function turnToRealData(){
```

Comment:
What is real data? If the purpose of this is to pull things out of local storage, don't you think the function name should communicate that?